packages/api-key-stamper/src/tink/elliptic_curves.ts

```diff
@@ -172,7 +172,7 @@ export function pointDecode(point: Uint8Array): JsonWebKey {
     point.length !== uncompressedLength
   ) {
     throw new Error(
-      "Invalid length: point is not in compressed or uncompressed format"
+      "Invalid API key: Ensure that you are using a valid public and private key for your API key"
```

Comment:
Sorry to make your life more difficult but I don't think this message should be updated. We're in `src/tink/elliptic_curves.ts` which deals with generic crypto primitives. It's also a vendored file which shouldn't be touched unless we really need to.

My proposal: let's look at the callers of `pointDecode`, catch errors, and re-throw with a different message?

For example, when this fails: https://github.com/tkhq/sdk/blob/d9a046573119022e202fdcbeac70d98b5d66c0a7/packages/sdk-browser/src/utils.ts#L34

This isn't related to API keys. It's because the targetPublicKey is invalid. So you can do something like:
```ts
try {
    const jwk = pointDecode(targetKeyBytes);
} catch (e: Error) {
    if (e.message === "Invalid length: point is not in compressed or uncompressed format") {
        // throw a more helpful error with context
        throw new Error(`target public key is not a valid compressed public key: ${targetPublicKey}`);
    } else {
        // rethrow unmodified
        throw e
    }
}

Reply:
Hm yeah, that seems like the way to handle it. I didnt realize this was a vendored file, we should def not change it in that case

Reply:
I added some try/catches to where decodePoint is being called in the code. However I didn't add the conditional to catch just that error message. All of the error messages that are thrown in decodePoint would be pretty unhelpful imo, so I added general error messages that will lead the user to solving their problem

Reply:
Interested in conflicting opinions on this!